zerver/webhooks/sentry/view.py

```diff
@@ -101,9 +108,11 @@ def handle_event_payload(event: Dict[str, Any]) -> Tuple[str, str]:
     if syntax_highlight_as == "":  # nocoverage
         logging.info("Unknown Sentry platform: %s", platform_name)
 
+    if is_sample_event(event):
+        pass
     # We shouldn't support the officially deprecated Raven series of
     # Python SDKs.
-    if platform_name == "python" and int(event["version"]) < 7:
+    elif platform_name == "python" and int(event["version"]) < 7:
         raise UnsupportedWebhookEventTypeError("Raven SDK")
```

Comment:
I think it would be clearer if the `is_sample_event` code were moved in here -- the only thing it's protecting against is the Python sample event having too early a version in it.  If we have to write custom acceptance code for other languages, we can pull it out then, but I'd prefer to limit checking this "magic tag" as much as possible.

```suggestion
    if platform_name == "python" and int(event["version"]) < 7:
        # The sample event is still an old "version" -- accept it even though we don't accept events from the old Python SDK:
        tags = event.get("tags", [])
        if ["sample_event", "yes"] not in tags:
            raise UnsupportedWebhookEventTypeError("Raven SDK")
```

Reply:
You are right, right now the check is for python only as for other samples the condition is already false. I will make the suggested changed and push it here shortly.